.circleci/config.yml

```diff
@@ -225,3 +225,13 @@ workflows:
               only: /^[0-9]+\.[0-9]+\.[0-9]+/
             branches:
               ignore: /.*/
+  main-branch:
+    jobs:
+      - test_android:
+          filters:
+            branches:
+              only: ci-browserstack-config
```

Comment:
This should be `main` right?

Reply:
yep, was just testing that it worked on this branch